### ISSUE TITLE
Question about using the data collection in realworld

### ISSUE THREAD

**Issue by Ju6276** [YanjieZe/3D-Diffusion-Policy#76]
Thanks for your work! You mention in your repo that using the D435 camera will give a very poor point cloud. Is it a fusion of multiple D435's or just a single one?

**Comment by YanjieZe** [comment]
We use the single camera. I think with multiple cameras and maybe some 3D reconstruction algorithms, D435 would be also helpful.

**Comment by Ju6276** [comment]
Thank you for your reply. Your algorithms make a lot of sense, and I would like to reproduce the task pour and then train a task myself. So I prepared L515 camera, Franka robotic arm, Allegro hand. I am very much looking forward to the next step.
Now I have more questions about the setup. I noticed that you guys wrapped blue tape (or something?)  around the fingers, white tape where it connects to the Franka, and a blue mat. I would like to ask about the purpose of this tape here. (In my opinion, Is this for the camera to better recognise finger joints?)
Also is the model at the base of the Allgreo hand 3d printed in PLA material? I'm a bit worried if it's solid enough.
<img width="544" alt="微信图片_20241018235020" src="https://github.com/user-attachments/assets/cbc716fb-2461-4856-8eea-76e9e035d183">

Thanks a lot for your time.
